source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
+        "region": "1",
+        "subregion": "2"
+      }
+    },
+    "application": "gfw",
+    "language": "en",
+    "env": "production",
+    "resource": {
+      "type": "EMAIL",
+      "content": "email@address.com"
     }
   }'
 ```
 
-#### Wood fiber
+A subscription can refer to a country, one of its regions, or a subregion within a region. Countries are identified by their ISO 3166-1 alpha-3 code - check [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) for a list of all the available country codes. Regions and subregions are identified by their respective GADM id, which can be obtained from GADM's dataset [here](https://gadm.org/data.html). When creating a subscription for a region, the country ISO must be specified. For subscribing to a subregion, both region and country ISO must be provided.
+
+Field                 | Description                                                    | Type
+--------------------- | :------------------------------------------------------------: | ----------------:
+`params.iso.country`  | ISO 3-letter code of the country to subscribe                  | String
+`params.iso.region`   | Region id to subscribe                                         | String
+`params.iso.subregion`| Subregion id to subscribe (optional)                           | String
+
+#### Subscribing to a protected area from World Database on Protected Areas
+
+> Creating a subscription providing a WDPA id in the params field:
 
 ```shell
-curl -X POST https://api.resourcewatch.org/v1/subscriptions \
+curl -X POST "https://api.resourcewatch.org/v1/subscriptions" \
 -H "Authorization: Bearer <your-token>" \
 -H "Content-Type: application/json"  -d \
  '{
-   "name": "<name>",
-   "application": "<application>",
-   "env": <environment>,
-   "language": "<language>",
-   "resource": {
-       "type": "<type>",
-       "content": "<content>"
-   },
-   "datasets" : ["<dataset>"],
-   "params": {
-       "use": "fiber",
-       "useid": <id>
+    "datasets": ["<dataset-id>"],
+    "params": {
+      "wdpaid": "32054"
     },
-	"datasetsQuery": [{}]
+    "application": "gfw",
+    "language": "en",
+    "env": "production",
+    "resource": {
+      "type": "EMAIL",
+      "content": "email@address.com"
+    }
   }'
 ```
 
-#### Congo Basin logging roads
+A subscription can refer to a specific protected area by the id of that area in the World Database on Protected Areas. If this is the case, you should provide the WDPA id in the `params.wdpaid` field when creating the subscription. IDs of protected areas worldwide can be obtained from the [Protected Planet website](https://www.protectedplanet.net/).
+
+Field                 | Description                                                    | Type
+--------------------- | :------------------------------------------------------------: | ----------------:
+`params.wdpaid`       | Id of the protected area in the WDPA                           | String
+
+#### Subscribing to a geostore
```

Comment:
Confused about why I can subscribe to both an area and a geostore. I thought geostores contained areas?

Reply:
It's the other way around - an **area** may (or may not) contain a geostore :) 

An **area** is a way of defining a geographical region (i.e. you can create an **area** for a geostore, ISO code, WDPA id, etc.). So, I think the idea here is that you can define in the subscription which specific geographical region you are subscribing to (i.e. geostore, ISO, WDPA, etc), or you can just reference an **area**.

Don't know if this helps to clarify your doubts :)